backend/package.go

```diff
@@ -123,7 +128,7 @@ func str2map(s string) map[string]string {
 		if len(kvParts) == 1 {
 			ret[key] = ""
 		} else {
-			ret[key] = strings.TrimSpace(kvParts[1])
+			ret[key] = url.QueryEscape(strings.TrimSpace(kvParts[1]))
```

Comment:
Ah, sorry!  I meant the inverse of this so that env vars may contain shell-unsafe chars or one of the split chars `[ ,:]`.